internal/usagestats/repositories.go

```diff
@@ -59,3 +63,76 @@ func GetRepositories(ctx context.Context, db database.DB) (*Repositories, error)
 
 	return &total, nil
 }
+
+func GetRepositorySizeHistorgram(ctx context.Context, db database.DB) ([]RepoSizeBucket, error) {
+	kb := int64(1000)
+	mb := kb * kb
+	gb := kb * mb
+
+	var sizes []int64
+	sizes = append(sizes, 0)
+	sizes = append(sizes, kb)
+	sizes = append(sizes, mb)
+	sizes = append(sizes, gb)
+	sizes = append(sizes, 5*gb)
+	sizes = append(sizes, 15*gb)
+	sizes = append(sizes, 25*gb)
+	sizes = append(sizes, 50*gb)
+	sizes = append(sizes, 100*gb)
+
+	var results []RepoSizeBucket
+
+	baseQuery := "select coalesce(Count(repo_size_bytes), 0) from gitserver_repos"
```

Comment:
```suggestion
	baseQuery := "select coalesce(count(repo_size_bytes), 0) from gitserver_repos"
```
idk why camelcase sql is super triggering